Tests/scripts/validate_files_structure.py

```diff
@@ -409,4 +409,4 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    main()
```

Comment:
not sure what is this change

Reply:
we started to add something and erased it...

Reply:
cool